packages/flutter/test/cupertino/date_picker_test.dart

```diff
@@ -11,6 +11,120 @@ import 'package:flutter_test/flutter_test.dart';
 const Offset _kRowOffset = Offset(0.0, -32.0);
 
 void main() {
+  testWidgets('time picker scrolls to 00:00:00 after reset', (WidgetTester tester) async {
+    const Duration animationTime = Duration(milliseconds: 200);
+    final CupertinoTimerPickerController controller = CupertinoTimerPickerController(
+        resetAnimationDuration: animationTime,
+    );
+    Duration duration;
```

Comment:
Would it make more sense to call this `currentPickerTime` or something similar?  Given the number of durations being thrown around in this PR, it's easy to get confused.